frontend/packages/console-dynamic-plugin-sdk/src/webpack/ConsoleRemotePlugin.ts

```diff
@@ -48,63 +54,55 @@ export class ConsoleRemotePlugin {
   }
 
   apply(compiler: webpack.Compiler) {
-    if (!compiler.options.output.enabledLibraryTypes.includes(remoteEntryLibraryType)) {
-      compiler.options.output.enabledLibraryTypes.push(remoteEntryLibraryType);
-    }
-
-    // Apply relevant webpack plugins
-    compiler.hooks.afterPlugins.tap(ConsoleRemotePlugin.name, () => {
-      new webpack.container.ContainerPlugin({
-        name: this.pkg.consolePlugin.name,
-        library: { type: remoteEntryLibraryType, name: remoteEntryCallback },
-        filename: remoteEntryFile,
-        exposes: this.pkg.consolePlugin.exposedModules || {},
-      }).apply(compiler);
-
-      new webpack.sharing.SharePlugin({
-        shared: sharedPluginModules,
-      }).apply(compiler);
+    const logger = compiler.getInfrastructureLogger(ConsoleRemotePlugin.name);
+    const publicPath = `/api/plugins/${this.pkg.consolePlugin.name}/`;
```

Comment:
Console has an option for a base context root, although we don't use it for OpenShift. Would this break that?

Reply:
This option means that all plugin assets will be loaded from `<ConsoleOrigin>/api/plugins/<PluginName>` so relative to Console application's protocol / domain / port at runtime. Not sure if that is safe enough in relation to changing base context root :confused:

Reply:
No, that will break context roots. We might need it for HAC as well :/

Reply:
https://github.com/openshift/console/blob/master/cmd/bridge/main.go#L69